app/main.py

```diff
@@ -16,4 +16,32 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+
+# "змінний"
+sorted_variables["mutable"].append([
+    "The Shawshank Redemption",
+    "The Lord of the Rings: The Return of the King",
+    "Pulp Fiction",
+    "The Good, the Bad and the Ugly",
+    "The Matrix"
+])
+
+sorted_variables["mutable"].append({
+    "John": 4,
+    "Sergio": 3
+})
+
+sorted_variables["mutable"].append({1, 2, 25})
+
+# "незмінний"
+sorted_variables["immutable"].append(777)
+sorted_variables["immutable"].append(3.14)
+sorted_variables["immutable"].append(False)
+sorted_variables["immutable"].append("Richard")
+sorted_variables["immutable"].append(("michel", "michel@gmail.com", "12345678"))
```

Comment:
You should use the variables above but not create new ones. For example
`
sorted_variables = {
    "mutable": [variable_1, variable_2],
    "immutable": [variable_2, variable_3],
}
`